### PR TITLE
Feature/not null or empty aux

### PR DESCRIPTION
**Pull request by arturobernalg**
another approach to
https://github.com/apache/commons-lang/pull/716

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/ObjectUtils.java [hunk 1]
```diff
@@ -1237,6 +1243,37 @@ public static String toString(final Object obj, final String nullStr) {
         return obj == null ? nullStr : obj.toString();
     }
 
+    /**
+     * <p>Validate that the specified argument object is neither {@code null}
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/lang3/ObjectUtils.java:1247]: Please close HTML tags. The FIRST sentence in a Javadoc does not need to be in a paragraph, it is implied.

File: src/main/java/org/apache/commons/lang3/ObjectUtils.java [hunk 2]
```diff
@@ -1237,6 +1243,37 @@ public static String toString(final Object obj, final String nullStr) {
...
+     * nor a length of zero (no elements); otherwise throwing an exception.
+     *
+     * <p>The message of the exception is &quot;The object is null or empty&quot;.</p>
+     *
+     * @param object the object to check
+     *
+     * @throws IllegalArgumentException if {@code array} is {@code null}
+     * @throws IllegalArgumentException if {@code array} is empty.
+     */
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/lang3/ObjectUtils.java:1256]: Javadoc Add at since 3.12.0 for new public and protected APIs.

**Comment by arturobernalg** [comment]
HI @garydgregory 

Fix javadoc.

**Comment by garydgregory** [comment]
Closing in favor of a different implementation in git master which mirror the style and names of java.lang.Objects.requireNon*() methods.

**Pull request closed without merging**